include/reporters/catch_reporter_console.hpp

```diff
@@ -258,11 +258,14 @@ namespace Catch {
             }
         }
         void lazyPrintRunInfo() {
+            // Usefulness of this message is questionable.
```

Comment:
These changes certainly seem outside the scope of whitespace changes.  What's the rationale for these?